lib/dishes.rb

```diff
@@ -0,0 +1,24 @@
+class Dishes
+  attr_reader :menu
+  def initialize
+    @menu = nil
+  end
+
```

Comment:
I see what you've saying about it being hard coded, but I did it the other way around and it made it much lengthier to test - this is definitely the right approach in terms of stages